### PR TITLE
Fix thundercore support.

### PR DESCRIPTION
**Pull request by ashwinrz**
Urgent fix to ensure that the thundercore logo is properly displayed as defined in the etherwallet-custom.less file.

### CODE REVIEW

**Comment by ashwinrz** [comment]
@gamalielhere can we merge this as soon as tests are completed? This is a somewhat urgent fix for us. Thanks in advance for the prompt reviews!

File: app/scripts/directives/walletDecryptDrtv.html [hunk 1]
```diff
@@ -59,7 +59,7 @@ <h4 translate="decrypt_Access">
...
            aria-flowto="aria4"
-           ng-show="ajaxReq.type=='ETH'||ajaxReq.type=='ETC'||ajaxReq.type=='ROPSTEN ETH'||ajaxReq.type=='RINKEBY ETH'||ajaxReq.type=='KOVAN ETH'||ajaxReq.type=='EXP'||ajaxReq.type=='UBQ'||ajaxReq.type=='POA'||ajaxReq.type=='TOMO'||ajaxReq.type=='ELLA'||ajaxReq.type=='EGEM'||ajaxReq.type=='CLO'||ajaxReq.type=='ETSC'||ajaxReq.type=='MUSIC'||ajaxReq.type=='GO'||ajaxReq.type=='EOSC'||ajaxReq.type=='AKA'||ajaxReq.type=='ESN'||ajaxReq.type=='PIRL'||ajaxReq.type=='ETHO'||ajaxReq.type=='ATH'||ajaxReq.type=='MIX'||ajaxReq.type=='ILT'||ajaxReq.type=='REOSC'">
+           ng-show="ajaxReq.type=='ETH'||ajaxReq.type=='ETC'||ajaxReq.type=='ROPSTEN ETH'||ajaxReq.type=='RINKEBY ETH'||ajaxReq.type=='KOVAN ETH'||ajaxReq.type=='EXP'||ajaxReq.type=='UBQ'||ajaxReq.type=='POA'||ajaxReq.type=='TOMO'||ajaxReq.type=='ELLA'||ajaxReq.type=='EGEM'||ajaxReq.type=='CLO'||ajaxReq.type=='ETSC'||ajaxReq.type=='MUSIC'||ajaxReq.type=='GO'||ajaxReq.type=='EOSC'||ajaxReq.type=='AKA'||ajaxReq.type=='ESN'||ajaxReq.type=='PIRL'||ajaxReq.type=='ETHO'||ajaxReq.type=='ATH'||ajaxReq.type=='MIX'||ajaxReq.type=='ILT'||ajaxReq.type=='REOSC'||ajaxReq.type=='THUNDERCORE'">
```
Comment (gamalielhere) [inline app/scripts/directives/walletDecryptDrtv.html]: Can you include a link to your PR in Trezor?

Comment (ashwinrz) [reply app/scripts/directives/walletDecryptDrtv.html]: Don't have one, I can remove this. Wasn't sure how this functionality worked anyway

Comment (ashwinrz) [reply app/scripts/directives/walletDecryptDrtv.html]: Had a question regarding the vintage vs. new release support. If I want the newer version of MEW to support Thunder, where do I submit a PR, or will this PR be reflected on the newer version of the site?

Comment (gamalielhere) [reply app/scripts/directives/walletDecryptDrtv.html]: you can make a PR here: https://github.com/MyEtherWallet/MyEtherWallet

Comment (ashwinrz) [reply app/scripts/directives/walletDecryptDrtv.html]: Great, thanks for the help!

File: app/scripts/directives/walletDecryptDrtv.html [hunk 2]
```diff
@@ -1204,6 +1204,20 @@ <h4 id="modalTitle" class="modal-title" translate="ADD_Radio_5_Path" style="marg
                </label>
             </div>
 
+            <div class="col-sm-4">
```
Comment (gamalielhere) [inline app/scripts/directives/walletDecryptDrtv.html:1207]: I'm assuming you have a ledger PR?

Comment (ashwinp-r) [reply app/scripts/directives/walletDecryptDrtv.html]: Yeah we're in the process of integration with them - not yet completed though

Comment (ashwinp-r) [reply app/scripts/directives/walletDecryptDrtv.html]: Lmk if you want me to delete this and re-submit this change in the future once our integration is completed.

Comment (ashwinrz) [reply app/scripts/directives/walletDecryptDrtv.html:1207]: Yup: https://github.com/LedgerHQ/ledger-app-eth/pull/60

**Comment by ashwinrz** [comment]
@gamalielhere Any update on whether this can be merged?

**Review by gamalielhere — APPROVED** [review]

**Pull request merged**